bin/generate_base_images.py

```diff
@@ -186,7 +177,7 @@ def _build_all(
     if args.python_version == "all":
         python_versions = SUPPORTED_PYTHON_VERSIONS
     else:
-        python_versions = {args.python_version}
+        python_versions = args.python_version
```

Comment:
Shouldn't we keep the set braces? I think it'll iterate through the characters in the string otherwise.